k2/csrc/rm_epsilon.cu

```diff
@@ -767,9 +767,9 @@ void ComputeEpsilonClosure(FsaVec &epsilon_fsa, FsaVec *closure_fsa,
   // We repeatedly call ComputeEpsilonClosureOneIter() until there is no further
   // change in the FsaVec (this can be by simple comparison on arcs vector,
   // since thanks to sorting the order is deterministic).
-  Array1<Arc> epsilon_fsa_arcs = epsilon_fsa.values;
+  Array1<Arc> &epsilon_fsa_arcs = epsilon_fsa.values;
```

Comment:
Actually, sorry, I realized this is not the source of the problem, I did not notice the statement below, `closure_fsa_arcs = cur_iter_closure_fsa.values;`
I will refactor a little for clarity though.